boxwise_flask/models.py

```diff
@@ -24,5 +25,54 @@ def __unicode__(self):
 
     @staticmethod
     def get_camps():
-        # SELECT distinct id, organisation_id FROM dropapp_dev.camps;
         return Camps.select().order_by(Camps.name)
+
+
+class Cms_Usergroups_Camps(db.Model):
+    camp_id = CharField()
+    cms_usergroups_id = CharField()
+
+    class Meta:
+        primary_key = CompositeKey('camp_id', 'cms_usergroups_id')
+
+    def __unicode__(self):
+        return self.name
+
+    @staticmethod
+    def get_camp(usergroup_id):
+        camp = (Cms_Usergroups_Camps
+            .select(Cms_Usergroups_Camps.camp_id)
+            .where(Cms_Usergroups_Camps.cms_usergroups_id == usergroup_id)
+            )
+        return camp
+
+
+class Cms_Users(db.Model):
+    id = CharField()
+    organisation_id = CharField()
+    naam = CharField()
+    name = CharField(column_name='naam')
+    email = CharField()
+    cms_usergroups_id = CharField()
+    valid_firstday = CharField()
+    valid_lastday = CharField()
+
+    def __unicode__(self):
+        return self.name
+
+    @staticmethod
+    def get_all_users():
+        return Cms_Users.select().order_by(Cms_Users.naam)
+
+    @staticmethod
+    def get_user(email):
+        q = Cms_Users.select().where(Cms_Users.email == email).get()
```

Comment:
peewee allows writing the shortcut `q = Cms_Users.get(Cms_users.email == email)`, too.
I'd prefer the shorter version though I can also see that the present version sounds more SQL-ish

Reply:
```user``` instead of ```get_user```?

Reply:
`user` instead of `q` (which is probably short for query, yet I'd consider that variable the result of a query rather)